frappe/printing/doctype/letter_head/letter_head.py

```diff
@@ -36,10 +36,27 @@ def set_image(self):
 					<img src="{self.image}" alt="{self.name}" {dimension}="{dimension_value}" style="{dimension}: {dimension_value}px;">
 				</div>
 				"""
-				frappe.msgprint(frappe._("Header HTML set from attachment {0}").format(self.image), alert=True)
+				frappe.msgprint(_("Header HTML set from attachment {0}").format(self.image), alert=True)
 			else:
 				frappe.msgprint(
-					frappe._("Please attach an image file to set HTML"), alert=True, indicator="orange"
+					_("Please attach an image file to set HTML for Letter Head."), alert=True, indicator="orange"
+				)
+
+		if self.footer_source == "Image":
+			if self.footer_image and is_image(self.footer_image):
+				self.footer_image_width = flt(self.footer_image_width)
+				self.footer_image_height = flt(self.footer_image_height)
+				dimension = "width" if self.footer_image_width > self.footer_image_height else "height"
```

Comment:
Could you add a comment explaining why we need only width if it's bigger, and only height if it's smaller or equal?